geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/repository/IndexRepositoryImpl.java

```diff
@@ -106,6 +112,9 @@ public void delete(Object key) throws IOException {
     try {
       Term keyTerm = SerializerUtil.toKeyTerm(key);
       writer.deleteDocuments(keyTerm);
+    } catch (Exception e) {
+      stats.incFailedEntries();
+      logger.info("Failed to delete index " + key + " due to " + e.getMessage());
```

Comment:
This should probably be "Failed to delete from index." Might want to mention which index the entry was not deleted from.

Do we even want a catch here, since we are not calling customer code.